lib/matplotlib/backends/backend_ps.py

```diff
@@ -1452,6 +1452,10 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     # multiple
     if sys.platform == 'win32': precmd = '%s &&'% os.path.splitdrive(tmpdir)[0]
     else: precmd = ''
+    #Replace \\ for / so latex does not think there is a function call
+    latexfile = latexfile.replace("\\", "/")
```

Comment:
As far as I understand the latex stuff, it is needed because otheriwese the next line would be interpreted as introducing a new directory "break":

`c:\something\short~1\tempfile` -> `c:/something/short~1/tempfile` -> `c:/something/short\\string~1/tempfile`

Without the first step, the dirs would read `c: - something - short - string~1 - tempfile`.

@Grillard  can you confirm that?